go/vt/sqlparser/ast.go

```diff
@@ -689,6 +690,30 @@ func (s *Signal) walkSubtree(visit Visit) error {
 	return nil
 }
 
+// Call represents the CALL statement
+type Call struct {
+	FuncName string
+	Params   []Expr
+}
+
+func (c *Call) Format(buf *TrackedBuffer) {
+	buf.Myprintf("call %s", c.FuncName)
+	if len(c.Params) > 0 {
+		buf.Myprintf("(")
+		for i, param := range c.Params {
+			if i > 0 {
+				buf.Myprintf(", ")
+			}
+			buf.Myprintf("%v", param)
+		}
+		buf.Myprintf(")")
+	}
+}
+
+func (c *Call) walkSubtree(visit Visit) error {
+	return nil
```

Comment:
Should walk Params here

Reply:
This doesn't actually matter for anything we do with vitess, but better to keep it consistent